xml/System.Drawing.Imaging.Effects/BlurEffect.xml

```diff
@@ -35,10 +35,21 @@
         <Parameter Name="expandEdge" Type="System.Boolean" />
       </Parameters>
       <Docs>
-        <param name="radius">To be added.</param>
-        <param name="expandEdge">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="radius">
+          <para>Real number that specifies the blur radius (the radius of the Gaussian convolution kernel) in pixels. The</para>
+          <para>radius must be in the range 0 through 256. As the radius increases, the resulting bitmap becomes more blurry.</para>
+        </param>
+        <param name="expandEdge">
+          <para>Boolean value that specifies whether the bitmap expands by an amount equal to the blur radius. If</para>
```

Comment:
```suggestion
          <para><see langword="true" /> to expand the bitmap by an amount equal to the blur radius so it can have soft edges. <see langword="false" /> to keep the bitmap the same size and clip the soft edges.</para></param>
```